src/components/overview/ProductInformation/AverageRating.jsx

```diff
@@ -10,12 +10,20 @@ const AverageRating = ({state, updateState}) => {
   }
   const avgRating = Math.round(ratingsSum / reviewCount * 10) / 10;
 
+  const scrollToReview = () => {
+    window.scrollTo({
+      top: 1500,
+      left: 0,
```

Comment:
Yes smooth scrolling!

Reply:
I'm curious about the top position '1500' - is that a number based on what you're seeing on your screen? I wonder if there's a way to get the exact location of the reviews module, because it may vary from screen to screen.

Reply:
I know we talked about setting an id and using the `href=#<id-name>` not sure if that can be incorporated here. Other ideas?

Reply:
I tried to test with different screen size but It worked fine! 

I used 1500 because the rating & review section is at the very bottom and when I set 1500, it always moves the screen to the bottom. And also, I chose this because scrollTo method gives smooth function! Let me know your thought.

Reply:
I used 1500 because the rating & review section is at the very bottom and I thought 1500 would be enough to moves the screen to the review section. And also, I chose this because scrollTo method gives smooth function.

I just tested if it moves to the review section, although the screen size get bigger when the user clicks the 'More Question Button. However, it moves to the bottom of the question section. 
so, I think it would be better to change 1500 to 2000 so that everytime the user clicks it even after clicking 'More Question Button', it moves to the very bottom on the page where the review section is. 
I tried this and it works fine. Let me know your thought!